src/parser/features/suggestions.rs

```diff
@@ -72,34 +72,88 @@ where
     }
 }
 
-#[cfg(all(test, features = "suggestions"))]
+#[cfg(all(test, feature = "suggestions"))]
 mod test {
     use super::*;
 
     #[test]
-    fn possible_values_match() {
+    fn missing_letter() {
         let p_vals = ["test", "possible", "values"];
-        assert_eq!(did_you_mean("tst", p_vals.iter()), Some("test"));
+        assert_eq!(did_you_mean("tst", p_vals.iter()), vec!["test".to_owned()]);
```

Comment:
Why the `.to_owned()` this doesn't seem necessary or am I missing something?

Reply:
its needed  in the flag case because of the custom type (tuple) and  I didn't pay attention and did it everywhere